src/main/resources/db/changelog/db.changelog-master.xml

```diff
@@ -10,4 +10,5 @@
     <include file="db/changelog/db.changelog-entities.xml"/>
     <include file="db/changelog/db.changelog-references.xml"/>
     <include file="db/changelog/db.changelog-indexes.xml"/>
+    <include file="db/changelog/db.changelog-defaultData.xml"/>
```

Comment:
hm.. not sure if it's a good name for a default step suggestions changelog - _db.changelog-defaultData.xml_ . Maybe we can name it _db.changelog-defaultStepSuggestions.xml_ or something like that?